.pre-commit-config.yaml

```diff
@@ -47,7 +47,6 @@ repos:
         language: docker_image
         entry: koalaman/shellcheck:v0.7.1
         files: ".sh"
-        args: ["--exclude=SC1004,SC1008,SC1009,SC1047,SC1050,SC1062,SC1071,SC1072,SC1090,SC1091,SC1105,SC1107,SC1123,SC1124,SC1126,SC2001,SC2002,SC2003,SC2004,SC2005,SC2006,SC2007,SC2009,SC2034,SC2035,SC2039,SC2155,SC2164,SC2181,SC2186,SC2196,SC2197,SC2230"]
```

Comment:
None of these rules fire anywhere, not sure why we have all of them excluded.

Reply:
I think these were the rules we did not feel were up to par for bento cause of inaccuracies etc.

Reply:
Do we still feel that way? For this project I think it makes sense to start fresh and slowly exclude rules we don't find useful vs. blanket exclude all these. Thoughts?